src/lumen/activity/football.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "lumen/activity/activity.hpp"
+#include "lumen/activity/field.hpp"
+#include "lumen/activity/team.hpp"
+
+namespace lumen::activity {
+
+/// Stores and displays the data for a Football game.
+class Football : public Activity {
+public:
+    Football();
+
+    /// Draw the fields to the display.
+    void update_display() override;
```

Comment:
A overly picky question: can we define getters and setters before other member functions?

Reply:
Shoots